homeassistant/components/energyid/quality_scale.yaml

```
@@ -0,0 +1,137 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: The integration does not expose any custom service actions.
+  appropriate-polling:
+    status: exempt
+    comment: The integration uses a push-based mechanism with a background sync task, not polling.
+  brands:
+    status: done
+  common-modules:
+    status: done
+  config-flow-test-coverage:
+    status: done
+  config-flow:
+    status: done
+  dependency-transparency:
+    status: done
+  docs-actions:
+    status: exempt
+    comment: The integration does not expose any custom service actions.
+  docs-high-level-description:
+    status: done
+  docs-installation-instructions:
+    status: done
+  docs-removal-instructions:
+    status: done
+  entity-event-setup:
+    status: exempt
+    comment: This integration does not create its own entities.
+  entity-unique-id:
+    status: exempt
+    comment: This integration does not create its own entities.
+  has-entity-name:
+    status: exempt
+    comment: This integration does not create its own entities.
+  runtime-data:
+    status: done
+  test-before-configure:
+    status: done
+  test-before-setup:
+    status: done
+  unique-config-entry:
+    status: done
+
+  # Silver
+  action-exceptions:
+    status: exempt
+    comment: The integration does not expose any custom service actions.
+  config-entry-unloading:
+    status: done
+  docs-configuration-parameters:
+    status: done
+  docs-installation-parameters:
+    status: done
+  entity-unavailable:
+    status: exempt
+    comment: This integration does not create its own entities.
+  integration-owner:
+    status: done
+  log-when-unavailable:
+    status: done
+    comment: The integration logs a single  message when the EnergyID service is unavailable.
+  parallel-updates:
+    status: exempt
+    comment: This integration does not create its own entities.
+  reauthentication-flow:
+    status: done
+  test-coverage:
+    status: done
+
+  # Gold
+  devices:
+    status: exempt
+    comment: The integration represents a service connection as a single device entry, created automatically by Home Assistant when the config entry is set up. This service device groups the mapped sensor entities but does not create individual device entries for each sensor as the sensors belong to their original integrations.
```

I think this is clearer.
```suggestion
    comment: The integration does not create any entities, nor does it create devices.
```